src/build/internal/webpack.ts

```diff
@@ -46,7 +48,7 @@ export const DEFAULT_BUILD_CONFIG: (x: {
     now: x.now,
     outDir: "dist",
     rootDir: x.rootDir,
-    sassVariableGetter: "getGlobal",
+    sassVariableGetter: defaultSassVariableGetter,
```

Comment:
I think this is an unintuitive change: someone who wants to use `getFoo("CONFIG.bar")` instead of `getGlobal("CONFIG.bar")` should be able to accomplish that like so:

```ts
    // …
    sassVariableGetter: "getFoo",
    // …
```

But as of the PR in its current state, they would have to write:

```ts
    // …
    sassVariableGetter: "getLol($keyString)",
    // …
```

Easily fixed, of course.

Reply:
I agree, the user shouldn't have to worry about the migration between `node-sass` and `dart-sass` the change should be transparent.

Reply:
I'm almost done with a fix for this.

Reply:
I have pushed a fix.